deployments/cluster/balancer.nix

```diff
@@ -83,6 +86,7 @@ in
 
   services.disciplina = let
     config-key = "alpha";
+    getSecret = k: ''"$(aws secretsmanager get-secret-value --secret-id ${env}/disciplina/cluster | jq -r .SecretString)"'';
```

Comment:
Is this function used at all? (I'm trying to understand what this key exactly serves for)

Reply:
No, not yet. It's part of the secrets management refactoring, which is WIP. I shouldn't have committed this.

Reply:
Anyway, what this does is insert a `"$(bash command here)"` into the invocation command, so that I can do things like `--committee-secret ${getSecret committee-secret}`, basically.